src/main/scala/GamePrompts.scala

```diff
@@ -0,0 +1,49 @@
+import GameHelpers.{calculateWordScore}
+
+object GamePrompts {
+
+  def promptName(): String = {
+    println("\u001b[33;1mWhat is your name?")
+    scala.io.StdIn.readLine().toLowerCase.capitalize
+  }
+
+  def promptLevel(): String = {
```

Comment:
It looks like there is a pattern of prompting with the `\u001b` colour and potentially asking for a response. Could we factor this out anywhere and avoid some of this repetition? Would it be worth it?
```scala
def notifyUser(message: String): Unit =
    println(s"\u001b[33;1m$question\u001b[0m ")

def promptUser(question: String): String = {
    notifyUser(question)
    scala.io.StdIn.readLine().toString
}
```